src/components/visualizations/geojson/geojson-visualization.js

```diff
@@ -56,7 +56,7 @@ export default class GeoJSONVisualization extends React.Component {
       bBox = GeoJsonUtils.isValidBBox(bBox) ? bBox : ohioBBox
 
       return (
-        <div className={isHidden ? 'hidden' : ''} test-id={`${source}-map`}>
+        <div data-testid={`${source}-map`} className={isHidden ? 'hidden' : ''} test-id={`${source}-map`}>
```

Comment:
why do we have `data-testid` and `test-id`?

Reply:
test-id was there to begin with. Enzyme uses test-id. I choose data-testid for the cypress test, because data-* is the strongest selector.